### ISSUE TITLE
draw on avatar - drawing swapped to a different avatar

### ISSUE THREAD

**Issue by helenvarley** [AUTUpStageTeam/UpStage#284]
today when vicki & i entered the walkthrough stage http://upstage.org.nz:8083/stages/walkthrough we discovered that the drawing she had done previously on her avatar was now showing up on mine, & there was a large dot drawn on her avatar instead.
"clear draw on avatar" did not work, we tried repeatedly.
eventually i cleared the stage from the stage edit page in the workshop, which forced a reload, and after this the drawings were gone from our avatars.

**Comment by helenvarley** [comment]
![screen shot 2015-08-05 at 8 37 49 am](https://cloud.githubusercontent.com/assets/3437324/9080658/4b56cc14-3b56-11e5-96bd-3567a8cd4fe2.png)
![screen shot 2015-08-05 at 8 36 18 am](https://cloud.githubusercontent.com/assets/3437324/9080659/4b609118-3b56-11e5-8305-78391c70648b.png)

**Comment by vickis** [comment]
My avatar had a large grey circle (could have been another user addition) BUT it seems weird that the drawing i had left from the walkthrough recently was attached to helen's av and not mine 
could it have been a remnant? artefact of the original drawing?

**Comment by helenvarley** [comment]
yes this was very wierd, that the drawing had transferred from vicki's av to mine. could it be connected to the mixing up of avatar names & voices? it seems that properties come unstuck from one avatar somehow. #228 #203 #230

**Comment by vickis** [comment]
Today i drew on avatar, i then cleared draw on avatar (right click menu) dropped the av (which seemed to be working for clear - and drew another image
The second image i drew attached to helen's avatar (this was very similar to the issue reported in the walkthrough stage 
![screen shot 2015-09-08 at 8 44 48 am](https://cloud.githubusercontent.com/assets/4249450/9730546/57c18036-566b-11e5-86b9-8f9fcf6639ff.png)
